test/system/220-healthcheck.bats

```diff
@@ -67,6 +67,9 @@ function _check_health {
     run_podman inspect $ctrname --format "{{.Config.HealthcheckOnFailureAction}}"
     is "$output" "kill" "on-failure action is set to kill"
 
+    run_podman inspect $ctrname --format "{{.Config.StartupHealthCheck.Test}}"
+    is "$output" "[CMD-SHELL /home/podman/healthcheck]" ".Config.StartupHealthCheck.Test"
```

Comment:
Suggestion for future (not a blocker, no need to repush): use `assert`, not `is`. The latter has too many subtle bugs and gotchas. I confirmed that none of them affect this instance, so, LGTM.